src/sentry/nodestore/bigtable/backend.py

```diff
@@ -248,8 +259,18 @@ def delete_multi(self, id_list):
             row.delete()
             rows.append(row)
 
-        self.connection.mutate_rows(rows)
-        self._delete_cache_items(id_list)
+        deleted_ids = []
+        errors = []
+        for id, status in zip(id_list, self.connection.mutate_rows(rows)):
+            if status.code == 0:
+                deleted_ids.append(id)
+            else:
+                errors.append(BigtableError(status.code, status.message))
+
+        self._delete_cache_items(deleted_ids)
```

Comment:
I have a suspicion there are circumstances where a delete might succeed be represented as a failure (e.g. network timeouts or failures to meet requested consistency levels for replication), whether or not this is the right approach to dealing with those kinds of failures will probably need to be re-evaluated at some point (same goes for single row deletion or any row mutation, I guess.)